WordPress/gutenberg
# Fix: Update styles for checkbox and radio controls.

## What?
Fixes issue: #61652 

## Why?
The current spacing between the checkbox input and its label is inconsistent with that of the radio buttons, even though they have the same size.

## How?
This PR updates the stylings for the checkbox control and radio control to make them consistent.

## Testing Instructions
1. Open any page or post in the editor and open its settings panel.
2. Click on the published or discussions buttons and another tab should appear where one can test the checkbox and radio controls simultaneously.

### Testing Instructions for Keyboard
nil

## Screenshots or screencast 
<img width="308" alt="image" src="https://github.com/WordPress/gutenberg/assets/55375170/3d6f0be5-2b02-4596-bd2e-8027298ef477">
